share/benchmark/xtp_benchmark.in

```diff
@@ -198,6 +231,60 @@ if OPTIONS.run:
     print("{:1.2f}% of runs completed successfully".format(
         float(goodruns)/float(len(moldata))*100.0))
 
-    print("Writing benchmark data to 'result.json'")
-    with open('result.json', 'w') as fp:
+    resultsfile=OPTIONS.output_folder+'/result.json'
+    if(os.path.isfile(resultsfile)):
+        with open(resultsfile, "r") as f:
+            oldresults=json.load(f)
+        oldresults.update(moldata)
+        moldata=oldresults
+        print(f"Updating benchmark data in '{resultsfile}'")
+    else:
+        print(f"Writing benchmark data to '{resultsfile}'")
+    with open(resultsfile, 'w') as fp:
         json.dump(moldata, fp, sort_keys=True, indent=4)
+
+
+if OPTIONS.analyze:
+    resultsfile=OPTIONS.output_folder+'/result.json'
+    reference=None
+    with open(resultsfile, "r") as f:
+            result=json.load(f)
+    
+    if OPTIONS.reference:
+        with open(OPTIONS.reference, "r") as f:
+            reference=json.load(f)
+
+    
+    def print_timing(molname,keyword):
+        new_value=float(result[molname][keyword])
+        if reference==None:
+            print(f"{keyword}:{new_value:.2f}")
+        else:
+            new_value=float(result[molname][keyword])
+            ref_value=float(reference[molname][keyword])
+            change=(new_value-ref_value)/ref_value
+            print(f"{keyword}: {new_value:.2f}\tref: {ref_value:.2f}\t change:{change*100:.2f}%")
+
+    def print_energy(molname):
+        new_value=float(result[molname]["S1[Hrt]"])
+        if reference==None:
```

Comment:
(I have my pedantic hat on), the "pytonic" way of checking if a variable is `None` is:
```py
if reference is None:
```